spec/models/apartments_spec.rb

```diff
@@ -1,7 +1,51 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
-  it("Apartment model exists") do
-  expect{Apartment.create}.to_not raise_error 
+
+  it 'is not valid without a street' do
+    Sza = Apartment.create street: '', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs.', bedrooms: '7', bathrooms: '7', image: "N/A", user_id: 'SzaSeason' 
```

Comment:
Just a note on paying attention to the casing for variables.  Variables should be inside of ruby should be in  snake_case, however when you are calling on a class like Apartment, it will be in PascalCase.